### DISCUSSION TITLE
Is there a setting to avoid replacing $$ with $ upon inserting a wildcard file's contents?

### DISCUSSION THREAD

**Discussion by Picometer12** [DominikDoom/a1111-sd-webui-tagcomplete#247] [Q&A]
Suppose I have a wildcard file whose only entry is a dynamic prompt:  `{A|B}, {0-3$$C|D|E}`

With "search for wildcards" enabled, TagComplete can find this file and insert the only row, but when it does it removes one of the `$`, yielding `{A|B}, {0-3$C|D|E}`

Is there a setting I'm missing to allow me to keep my dynamic prompt in tact?

**Comment by DominikDoom** [comment]
This is unintended, there likely is something in the sanitization logic that escapes it accidentally. I'll see if I can fix it.

**Reply by DominikDoom** [reply]
Since it's a bug I have created issue #248 to track it.

**Comment by DominikDoom** [comment]
The bug should now be fixed as of a7233a5. It was caused by the replace functions treating $ as a placeholder and $$ for the actual $ character, so in the dynamic prompt wildcards they were interpreted as a single one.
